forest/src/cli/chain_cmd.rs

```diff
@@ -32,9 +37,9 @@ pub enum ChainCommands {
         /// Skip old messages
         #[structopt(short)]
         skip_old_messages: bool,
-        /// Snapshot output path. Default to `forest_snapshot_{year}-{month}-{day}_height_{height}.car`
-        #[structopt(short)]
-        output_path: Option<String>,
+        /// Snapshot output path. Default to [OUTPUT_PATH_DEFAULT_VALUE]`
```

Comment:
We should put in documentation with what "keywords" the path can be formatted and what they represent (e.g. clearly state whether the time is UTC)